Appraisals

```diff
@@ -501,6 +501,7 @@ elsif Gem::Version.new('2.3.0') <= Gem::Version.new(RUBY_VERSION) \
       gem 'grape'
       gem 'graphql'
       gem 'grpc'
+      gem 'google-protobuf', '~> 3.11.0' # Last version to support Ruby < 2.5
```

Comment:
We could lock `grpc` to a very old version, but instead we can lock it's transitive dependency `google-protobuf`, to the latest supported version, which is closer to what clients using these older versions of Ruby are likely doing.
This allows for newer `grpc` installations, backed by an older `google-protobuf`.